Graphics/HLSL2GLSLConverterLib/src/HLSL2GLSLConverterImpl.cpp

```diff
@@ -3065,7 +3065,7 @@ void HLSL2GLSLConverterImpl::ConversionStream::ProcessFunctionParameters(TokenLi
                     // ^
                     ParamInfo.GSAttribs.PrimType = ShaderParameterInfo::GSAttributes::PrimitiveType::TriangleAdj;
                     ++Token;
-
+                // fall through
```

Comment:
This is actually a bug...

Reply:
Yeah, that's exactly why my company has that flag on by default, you can make the compiler ignore it if you put a comment like this there.

Reply:
I honestly don't understand why it didn't get caught. We [use `-Werror -Wall -Wextra` plus few other additional warnings](https://github.com/DiligentGraphics/DiligentCore/blob/master/CMakeLists.txt#L325).